app/services/create_actual.rb

```diff
@@ -11,8 +9,14 @@ def initialize(activity:, report: nil, user: nil)
   def call(attributes: {})
     actual.parent_activity = activity
 
-    body = attributes.delete(:comment)
-    actual.build_comment(body: body, commentable: actual, report: @report) if body.present?
+    comment_body = attributes.delete(:comment).to_s.strip
+    if comment_body.present?
+      actual.build_comment(
+        body: comment_body,
+        commentable: actual,
```

Comment:
do we need to specify `commentable: actual` if we're also doing `actual.build_comment`

Reply:
It looks like we do, when the actual itself isn't persisted yet, so the `commentable_id` doesn't end up `nil`.

Reply:
😣  ok